src/dahua.ts

```diff
@@ -28,8 +26,12 @@ class DahuaEvents {
     public RECONNECTING_EVENT_NAME: string = 'reconnecting'
 
     private host:                   string
+    private eventsWatchUri:         string
 
-    constructor(host: string, user: string, pass: string, useHttp: boolean) {
+    constructor(host: string, user: string, pass: string, useHttp: boolean, events: Set<string>) {
+        //cgi-bin/eventManager.cgi?action=attach&codes=[AlarmLocal,VideoMotion,VideoLoss,VideoBlind]
+        //See more https://github.com/SaWey/home-assistant-dahua-event
+        this.eventsWatchUri = `/cgi-bin/eventManager.cgi?action=attach&codes=[${Array.from(events).join(",")}]`;

```

Comment:
Can we keep this as the EVENTS_URI const? 
`/cgi-bin/eventManager.cgi?action=attach&codes=`

Reply:
I don't know what it would facilitate, still can update if it matters. We construct Uri parameters without URI builder abstraction for events, so it requires explicit knowledge of where/what to inject. I.e. `/cgi-bin/eventManager.cgi' could be const, action/codes added as parameters in constructor.